clients/http-client-common/src/main/java/org/apache/servicecomb/http/client/event/RefreshEndpointEvent.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.http.client.event;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class RefreshEndpointEvent {
+
+  private static final String SAME_ZONE = "sameZone";
+
+  private static final String SAME_REGION = "sameRegion";
+
+  private Map<String, List<String>> zoneAndRegion;
+
+  private String name;
+
+  public RefreshEndpointEvent(Map<String, List<String>> zoneAndRegion, String name) {
```

Comment:
event do not group address, only need to add List<EndpointAddress> or List<AddressInfo>

Reply:
I think there is little difference between packaging and disassembly, and disassembly is not conducive to expansion